official/resnet/cifar10_test.py

```diff
@@ -157,13 +168,27 @@ def test_cifar10model_shape_v2(self):
   def test_cifar10_end_to_end_synthetic_v1(self):
     integration.run_synthetic(
         main=cifar10_main.run_cifar, tmp_root=self.get_temp_dir(),
-        extra_flags=['-resnet_version', '1']
+        extra_flags=['-resnet_version', '1', '-batch_size', '4']
     )
 
   def test_cifar10_end_to_end_synthetic_v2(self):
     integration.run_synthetic(
         main=cifar10_main.run_cifar, tmp_root=self.get_temp_dir(),
-        extra_flags=['-resnet_version', '2']
+        extra_flags=['-resnet_version', '2', '-batch_size', '4']
+    )
+
+  def test_cifar10_end_to_end_keras_synthetic_v1(self):
```

Comment:
Do these tests run with 0 or 1 num_gpus or both?

Reply:
It will just take the default for the flag. Which I believe is 1 if a GPU is available else 0.